web/admin/application/languages/es_ES/es_ES.po

```diff
@@ -2413,8 +2414,9 @@ msgstr ""
 msgid "White Lists"
 msgstr "Listas blancas"
 
+#, fuzzy
 msgid "Wholesale"
-msgid_plural "Wholesale"
+msgid_plural "Wholesales"
 msgstr[0] "Wholesale"
 msgstr[1] "Wholesale"
 
```

Comment:
Same here, _Wholesales_, like _friends_ and _Retails_ it's a technical name not translated:

```
 msgid "Wholesale"
 msgid_plural "Wholesales"
 msgstr[0] "Wholesale"
 msgstr[1] "Wholesales"
```